packages/web-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -52,7 +48,7 @@ const _Checkbox = (props: SpiritCheckboxProps, ref: ForwardedRef<HTMLInputElemen
         ref={ref}
       />
       <span className={classProps.text}>
```

Comment:
```suggestion
      <div className={classProps.text}>
```

There can't be `span` because  `HelperText` and `ValidationText` accept nodes; we need there `div`